src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -972,10 +980,24 @@ export function getVirtualStudyDescription(
                     genomicDataFilter.profileType
                 );
                 const name = attributeNamesSet[uniqueKey];
+                const dataFilterValues = genomicDataFilter.values;
+                const dataType = getDataTypeByDataFilterValues(
+                    dataFilterValues
+                );
+
                 if (name) {
                     filterLines.push(
                         `- ${name}: ${intervalFiltersDisplayValue(
-                            genomicDataFilter.values,
+                            dataType === DataType.NUMBER
```

Comment:
i believe that the interval filters can be edited by clicking on the filter pills. can you check see how this is affected?